### DISCUSSION TITLE
Apply ktlint to Github Action

### DISCUSSION THREAD

**Discussion by zion830** [euphony-io/euphony#227] [Ideas]
The percentage of Kotlin code is low yet, I suggest applying it later.

## Official Guide
* https://github.com/ScaCap/action-ktlint
* https://github.com/marketplace/actions/run-ktlint-with-reviewdog

## How it works
Whenever a PR is opened, Github Action automatically checks or reviews `.kt` files whether they follow the Kotlin convention well or not.

![image](https://user-images.githubusercontent.com/35446215/188463574-344a0840-a1d9-430c-b991-a7931435eb8f.png)

**Comment by designe** [comment]
It is like a auto lint bot for kotlin!
Great proposal :) Do it!

if you need some secret keyfor this, please let me know :)

**Comment by yeonns** [comment]
@zion830 So nice :+1: 
@zion830 @designe I read `ScaCap/action-ktlint` and I think `android` option is also useful. 
- android
  - Optional. Runs KtLint with [Android Kotlin Style Guide](https://android.github.io/kotlin-guides/style.html) [true/false] The default value is false.

What do you think about applying this option? :eyes:

**Reply by designe** [reply]
@yeonns @zion830 
If this android option is activated, I think action will not make any other comments kk
I like both options. kotlin code is starting point. so, this is up to us. we can select :)

**Reply by zion830** [reply]
I also think android option is useful!
